src/Pages/PlayQuiz/Question/Question.tsx

```diff
@@ -0,0 +1,242 @@
+import classes from './Question.module.css'
+import {useState,useEffect, SyntheticEvent} from 'react'
+import {Question as QuestionType} from '../../../Store/QuizContext/QuizContext.types'
+import {FormGroup,FormControlLabel,Checkbox,RadioGroup,Radio,Button} from '@material-ui/core'
+import {KeyboardArrowRight,SaveAlt} from '@material-ui/icons'
+import {useQuiz} from '../../../Store/QuizContext/QuizContext'
+import { useAuth } from '../../../Store/AuthContext/AuthContext'
+import {usePlayQuiz} from '../../../Store/PlayQuizContext/PlayQuizContext'
+import {useHistory} from 'react-router-dom'
+
+interface QuestionProps extends QuestionType{
+    totalQuestions:number;
+    currentIndex:number
+}
+
+export const Question=({id,multipleCorrect,options,points,question,hint,negativePoints,totalQuestions,currentIndex}:QuestionProps)=>{
+    const [timer,setTimer]=useState(30)
+    const [checked,setChecked]=useState<string[]>([])
+    const [showHint,setShowHint]=useState(false)
```

Comment:
I don't see the hint in the app.

Reply:
Hints can be there if the quiz creator chooses to add them for the question